### PR TITLE
Feature/documentation

### PR DESCRIPTION
**Pull request by johnkrovitch**

### CODE REVIEW

File: Action/Configuration/ActionConfiguration.php [hunk 1]
```diff
@@ -100,7 +100,7 @@ public function configureOptions(OptionsResolver $resolver)
                 'xls'
             ]);
 
-        // entity will be retrived with this order. It should be an array of field/order mapping
+        // entity will be retrieved with this order. It should be an array of field/order mapping
```
Comment (johnkrovitch) [inline Action/Configuration/ActionConfiguration.php]: retrieved entities are sorted according to this option
it should be an array indexed by the field name with its order as a value

File: Application/Configuration/ApplicationConfiguration.php [hunk 2]
```diff
@@ -40,7 +40,7 @@ public function configureOptions(OptionsResolver $resolver)
 
         // main base template
         // as bundles are not loaded when reading the configuration, the kernel locateResources will always failed.
-        // So we must not check resource existance here.
+        // So we must not check resource existence here.
         $resolver->setDefault('base_template', 'LAGAdminBundle::admin.layout.html.twig');
```
Comment (johnkrovitch) [inline Application/Configuration/ApplicationConfiguration.php]: template or layout

File: README.md [hunk 3]
```diff
@@ -1,183 +1,218 @@
+[![Build Status](https://travis-ci.org/larriereguichet/AdminBundle.svg?branch=master)](https://travis-ci.org/larriereguichet/AdminBundle)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/larriereguichet/AdminBundle/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/larriereguichet/AdminBundle/?branch=master)
+[![SensioLabsInsight](https://insight.sensiolabs.com/projects/c8e28654-44c7-46f3-9450-497e37bda3d0/mini.png)](https://insight.sensiolabs.com/projects/c8e28654-44c7-46f3-9450-497e37bda3d0)
+
+
 # AdminBundle
 
-AdminBundle allows you to create flexible and robust management application, using a simple configuration,
+AdminBundle allows you to create flexible and robust backoffice application, using a simple yml configuration,
```
Comment (johnkrovitch) [inline README.md]: The AdminBundle allows to create a flexible and robust backoffice on any Symfony application, with simple `yml` configuration only.

File: README.md [hunk 4]
```diff
@@ -1,183 +1,218 @@
...
 for your Symfony application.
 
-Development and Documentation are in progress
+AdminBumble creates Admin entities (like 'product' or 'store'), to handle your Doctrine entities CRUD views. Each Admin
```
Comment (lutangar) [inline README.md]: Ease the handling of CRUD views by configuring `Admin` objects on top of your Doctrine entities. Each `Admin` has one or many `Action`. 
By default, the four actions of a classical CRUD are available (`create`, `edit`, `delete` and `list`) and the creation of new actions is painless.

File: README.md [hunk 5]
```diff
@@ -1,183 +1,218 @@
...
+has Actions entities (like 'create' or 'list'). By default, the four actions of the CRUD (create, edit, list and delete)
+are available by default, but you can add your own actions. 
 
-Testing : 
-branch dev [![Build Status](https://travis-ci.org/larriereguichet/AdminBundle.svg?branch=dev)](https://travis-ci.org/larriereguichet/AdminBundle)
-coverage [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/larriereguichet/AdminBundle/badges/quality-score.png?b=dev)](https://scrutinizer-ci.com/g/larriereguichet/AdminBundle/?branch=dev)
+If you require more flexibility, you can easily override any part of the process (repositories, controllers, views...).
+The purpose of the bundle is to provide an Admin interface with default configuration, and allows the user to add his
+specific need where he wants.
```
Comment (lutangar) [inline README.md]: , and allow to implements any specific needs without any hassle.

File: README.md [hunk 6]
```diff
@@ -1,183 +1,218 @@
...
+    use LAG\AdminBundle\Repository\DoctrineRepository;
 
+    class PlanetRepository extends DoctrineRepository {
+    
+    ...
+
+```
 
-__Configuration Reference__
+Now your admin is ready, but as new routes will be generated, you should clear the Symfony's cache.
```
Comment (lutangar) [inline README.md]: Your admin is now ready!

> As new routes will be generated, you should clear the Symfony's cache.

**Pull request merged**